js-cookie/js-cookie.d.ts

```diff
@@ -0,0 +1,73 @@
+// Type definitions for js-cookie v2.0
+// Project: https://github.com/js-cookie/js-cookie
+// Definitions by: Theodore Brown <https://github.com/theodorejb>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+declare module Cookies {
+    interface CookieOptions {
```

Comment:
@theodorejb 

The "options" naming was changed to "attributes": https://github.com/js-cookie/js-cookie/commit/6ff48c80cbbf9ed6e0e6bf1757a8890f16970cfd